File: interpreters/game.py
# Python3 translation files

class Game:

    def __init__(self):
        pass

    def loadLevel(self):
        return engine.send(f'game.levelLoader.load('{name}')')


class Character:
    def __init__(self, x, y):
        engine.send(f'game.level.addElement("character", {x}, {y})')

File: interpreters/game.rb
#
# Ruby translation files for the game
#

# Allows user to interact with game
class Game

  # Loads a level in the game
  def load_level name
    engine.send "game.levelLoader.load('#{name}')"
  end

  # Reset the game
  def reset
    engine.send "game.rest"
  end

end

# Allows to interact with the character
class Character

  def new x, y
    engine.send "game.level.addElement('character', #{x}, #{y})"
  end
end